### PR TITLE
Implemented tab drag and drop functionality

### PR DESCRIPTION
**Pull request by soumyamahunt**
Implemented tab drag and drop functionality across app instances, dragging tab outside creates a new instance of the app.

4mvKFW6UCI

### CODE REVIEW

**Comment by Jaiganeshkumaran** [comment]
Great!

**Review by jaigak — APPROVED** [review]

File: Files/Strings/en-US/Resources.resw [hunk 1]
```diff
@@ -837,4 +837,7 @@
   <data name="syncStatusColumn.Header" xml:space="preserve">
     <value>Status</value>
   </data>
+  <data name="TabStripDragAndDropUIOverrideCaption" xml:space="preserve">
+    <value>Move tab here</value>
```
Comment (tsvietOK) [inline Files/Strings/en-US/Resources.resw:841]: I think it is better to use "Move" or "Move tab"

**Comment by yaichenbaum** [comment]
@soumyamahunt What a great pull request! Can you please target the vertical tabs branch where we have our newest version of the tabbed ui?

**Comment by lukeblevins** [comment]
@soumyamahunt I second what @yair100 said. These are good changes, and I'd hate to see them get lost after we announce future improvements to this space

**Comment by soumyamahunt** [comment]
@yair100 @duke7553 I am closing this PR since I have implemented drag-drop feature based on the new vertical tabs branch. I will submit new PR shortly.

**Pull request closed without merging**